selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/SeleniumWebDriverHelper.java

```diff
@@ -148,6 +151,52 @@ public WebElement waitVisibility(WebElement webElement) {
     return waitVisibility(webElement, DEFAULT_TIMEOUT);
   }
 
+  /**
+   * Waits until {@link WebElement} with provided locator {@link By} be invisible
+   *
+   * @param elementLocator
+   * @param timeout waiting time in seconds
+   * @return {@code true} - if not visible,
```

Comment:
returning `boolean` type is useless here because it can only return **true**, or throw timeout exception.

Reply:
In this case all usages of this method need check with Assert.assertTrue()